eng/packaging.targets

```diff
@@ -124,6 +124,8 @@
                                    Condition="'$(IncludeSymbols)' == 'true' and Exists('%(_RuntimeSymbolPath.Identity)')" />
     </ItemGroup>
   </Target>
+  
+  <Target Name="Pack" Condition="'$(IsSourceProject)' != 'true'" />
```

Comment:
I noticed we were packing the test projects and uploading to the artifacts. I don't think we care about them. Do we?

If we don't care about them, then is this the right way to skip test project packiung, @ViktorHofer @ericstj? 

<details><summary>Before:</summary>

Example: https://dev.azure.com/dnceng-public/public/_build/results?buildId=870307&view=artifacts&pathAsName=false&type=publishedArtifacts

Before
```
 The package Microsoft.SqlServer.TDS.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.SqlServer.TDS.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.SqlServer.TDS.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.IO.Redist.6.1.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.IO.Redist.6.1.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Json.4.8.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Reflection.DispatchProxy.4.8.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Reflection.DispatchProxy.4.8.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Json.4.8.1-ci.24564.1.symbols.nupkg'.
  The package Circle.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.nupkg'.
  The package Shapes.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Shapes.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.IO.Redist.6.1.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Json.4.8.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Reflection.DispatchProxy.4.8.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Reflection.DispatchProxy.4.8.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Json.4.8.1-ci.24564.1.symbols.nupkg'.
  The package Circle.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.nupkg'.
  The package Shapes.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Shapes.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Reflection.DispatchProxy.4.8.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Json.4.8.1-ci.24564.1.symbols.nupkg'.
  The package Circle.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.nupkg'.
  The package Shapes.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Shapes.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Shapes.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.nupkg'.
  The package Shapes.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Shapes.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Shapes.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Circle.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Shapes.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Threading.Tasks.Extensions.4.6.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Threading.Tasks.Extensions.4.6.1-ci.24564.1.symbols.nupkg'.
  The package Address.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  The package Address.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Address.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Address.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Numerics.Vectors.4.6.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Numerics.Vectors.4.6.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Xml.XPath.XmlDocument.4.7.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Xml.XPath.XmlDocument.4.7.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.Bcl.HashCode.6.0.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.Bcl.HashCode.6.0.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Buffers.4.6.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Buffers.4.6.1-ci.24564.1.symbols.nupkg'.
  The package Utf8String.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Utf8String.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Utf8String.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Net.WebSockets.WebSocketProtocol.5.1.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Net.WebSockets.WebSocketProtocol.5.1.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Memory.4.6.1-ci.24564.1.nupkg'.
  The package Microsoft.SqlServer.TDS.EndPoint.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.SqlServer.TDS.EndPoint.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Memory.4.6.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.SqlServer.TDS.EndPoint.1.0.0-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Data.SqlClient.4.9.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Runtime.CompilerServices.Unsafe.6.1.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Runtime.CompilerServices.Unsafe.6.1.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Data.SqlClient.4.9.1-ci.24564.1.symbols.nupkg'.
  The package Microsoft.SqlServer.TDS.Servers.1.0.0-ci.24564.1 is missing a readme. Go to https://aka.ms/nuget/authoring-best-practices/readme to learn why package readmes are important.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.SqlServer.TDS.Servers.1.0.0-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.SqlServer.TDS.Servers.1.0.0-ci.24564.1.symbols.nupkg'.
```
</details>

<details><summary>After:</summary>

After
```
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.IO.Redist.6.1.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.IO.Redist.6.1.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Numerics.Vectors.4.6.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Numerics.Vectors.4.6.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Json.4.8.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Json.4.8.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Threading.Tasks.Extensions.4.6.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Threading.Tasks.Extensions.4.6.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Buffers.4.6.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Buffers.4.6.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.Bcl.HashCode.6.0.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\Microsoft.Bcl.HashCode.6.0.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Reflection.DispatchProxy.4.8.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Reflection.DispatchProxy.4.8.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Xml.XPath.XmlDocument.4.7.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Xml.XPath.XmlDocument.4.7.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Memory.4.6.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Memory.4.6.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Runtime.CompilerServices.Unsafe.6.1.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Runtime.CompilerServices.Unsafe.6.1.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Data.SqlClient.4.9.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Net.WebSockets.WebSocketProtocol.5.1.1-ci.24564.1.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Net.WebSockets.WebSocketProtocol.5.1.1-ci.24564.1.symbols.nupkg'.
  Successfully created package 'C:\Users\calope\source\repos\maintenance-packages\artifacts\packages\Release\Shipping\System.Data.SqlClient.4.9.1-ci.24564.1.symbols.nupkg'.
```

</details>

Reply:
Submitted https://github.com/dotnet/maintenance-packages/pull/181 to fix this for real